cartoframes/io/managers/gbq_manager.py

```diff
@@ -0,0 +1,176 @@
+import os
+import math
+
+from google.cloud import bigquery
+from google.oauth2.credentials import Credentials
+
+from ...utils.logger import log
+from ...utils.utils import dtypes2vl, create_hash
+
+GEOID_KEY = 'geoid'
+GEOM_KEY = 'geom'
+MVT_DATASET = 'mvt_pool'
```

Comment:
We're using just a single table. If the user wants to visualize more than 1 query I think that's not compatible. @Jesus89 could you confirm? Just to add it to the backlog, not a priority, but just to have it on the radar.

Reply:
A new MVT table is created for a new query because the name of the table is a hash of the query. Multilayer works the same in CARTOframes, so it's possible to have more than one GBQ sources.